### DISCUSSION TITLE
Doc explaining difference between API endpoints between 2.* release and 4.0.4 release

### DISCUSSION THREAD

**Discussion by imaskm** [quay/clair#1235] [Q&A] [answered]
Hi, Is there any documentation which explains the change log in new version of API in release 4.0.4, as I was using 2.* version before and after upgrade to 4.0.4, I am getting 404 (not found) error for APIs:

e.g this api - http://clairsvc.ashwani:6060/v1/layers, is returning 404

We were using this version previously:
https://pkg.go.dev/github.com/coreos/clair@v0.0.0-20180717235726-b1cd09231990

which version of golang package of clair should I use which is compatible with 4.0.4 version as the latest above package is also pretty old.

**Comment by ldelossa** [comment, marked as answer]
Our v4 api is here: 

https://quay.github.io/clair/reference/api.html

**Reply by hdonnay** [reply]
For more context, the APIs are completely different and incompatible.
